### PR TITLE
Endofday Bug in ifelse Statement

### PR DESCRIPTION
**Pull request by David2464**
Pull Request Label
Bug
Team Member(s) who solved this issue
David Nellis
Summary
Extra semi-colon present which resulted in breaking the entire if statement. It has been removed.

Checklist from Issue of Things Solved
 - Removed semi-colon

### CODE REVIEW

File: code/processes/rdb.q [hunk 1]
```diff
@@ -107,7 +107,7 @@ endofday:{[date;processdata]
 	writedown[hdbdir;date];
         /-creates new changeset if this is a finspace application
         if[.finspace.enabled;
-                    changeset:.finspace.createchangeset[.finspace.database];
+		    changeset:.finspace.createchangeset[.finspace.database];
```
Comment (jonathonmcmurray) [inline code/processes/rdb.q:110]: indent has been (partially) changed from spaces to tabs, can you restore that to how it was before please?

File: code/processes/rdb.q [hunk 2]
```diff
@@ -121,7 +121,7 @@ endofday:{[date;processdata]
 	hdbs:distinct raze {exec w from .servers.getservers[x;y;()!();1b;0b]}'[`proctype`procname;(hdbtypes;hdbnames)];
 	$[.finspace.enabled;
                      .finspace.notifyhdb[;changeset] each .finspace.hdbclusters;
-                     notifyhdb[;date] each hdbs;
+		     notifyhdb[;date] each hdbs
```
Comment (jonathonmcmurray) [inline code/processes/rdb.q:124]: indent changed on this line as well

**Comment by David2464** [comment]
That should be that fixed now.

Thanks,
David


​

David Nellis
Graduate kdb developer

​
​
​
 
UK / Ireland
​Sturgeon Building
​Suite 5
​9 - 15 Queen Street
​Belfast, BT1 6EA
United States
Merrill Lynch Building 
​21st Floor, Suite 21099
​101 Hudson Street
​Jersey City, NJ 07302
Hong Kong
6 / F Alexandra House
​18 Chater Road 
​Central 
​Hong Kong
Singapore
7 - 12 Manhattan House 
​151 Chin Swee Road 
​Singapore 
​169876

​This email, its contents and any files attached are a confidential communication and are intended only for the named addressees indicated in the message.
If you are not the named addressee or if you have received this email in error, you may not, without the consent of Data Intellect, copy, use or rely on any information or attachments in any way. Please notify the sender by return email and delete it from your email system.
Unless separately agreed, Data Intellect does not accept any responsibility for the accuracy or completeness of the contents of this email or its attachments. Please note that any views, opinion or advice contained in this communication are those of the sending individual and
​not those of Data Intellect and Data Intellect shall have no liability whatsoever in relation to this communication (or its content) unless separately agreed.
This e-mail message is intended to be received only by persons entitled to receive the confidential information it may contain. E-mail messages to clients of Data Intellect may contain information that is confidential and legally privileged. Please do not read, copy, forward, or
​store this message unless you are an intended recipient of it. If you have received this message in error, please forward it to the sender and delete it completely from your computer system.
​
________________________________

File: code/processes/rdb.q [hunk 3]
```diff
@@ -121,7 +121,7 @@ endofday:{[date;processdata]
 	hdbs:distinct raze {exec w from .servers.getservers[x;y;()!();1b;0b]}'[`proctype`procname;(hdbtypes;hdbnames)];
 	$[.finspace.enabled;
                      .finspace.notifyhdb[;changeset] each .finspace.hdbclusters;
-                     notifyhdb[;date] each hdbs;
+	             notifyhdb[;date] each hdbs
```
Comment (jonathonmcmurray) [inline code/processes/rdb.q:124]: indentation is still partially changed here (now has one tab at start of line, previously was all spaces), if you can change that back please

Comment (David2464) [reply code/processes/rdb.q]: I've changed it again now. I just changed it entirely to spaces in Fixing indentation #3. Hopefully thats it now

**Review by jonathonmcmurray — APPROVED** [review]
LGTM

**Pull request merged**